frontends/mit-open/src/page-components/Header/MenuButton.tsx

```diff
@@ -23,22 +23,32 @@ const MenuButtonInner = styled.div({
   alignItems: "flex-start",
 })
 
-const StyledMenuButton = styled(Button)({
+const StyledMenuButton = styled.button(({ theme }) => ({
   padding: "0",
   background: "transparent",
   "&:hover:not(:disabled)": {
     background: "transparent",
   },
   touchAction: "none",
-})
+  textAlign: "center",
+  display: "inline-flex",
+  justifyContent: "center",
+  alignItems: "center",
+  color: theme.palette.text.primary,
+  transition: `background ${theme.transitions.duration.short}ms`,
+  cursor: "pointer",
+  borderWidth: "1px",
+  borderColor: "currentcolor",
```

Comment:
Could you remove these? They do nothing with `borderStyle: "none"`.

*In `<Button />` they do have an effect, but then `variant="text"` set borderStyle to none, so they aren't relevant for that variant.*